### ISSUE TITLE
Show the roll result to the user

### ISSUE THREAD

**Issue by Stanislavsonder** [Stanislavsonder/fate-core#8]
When the dice are rolled, the user should be able to see the summarized result of the roll (and the particular dice result).
The result should also sync to an a11y label and work with any other dice type in the future.

**Comment by Stanislavsonder** [comment]
Closed in #24
